src/main/java/com/sayweb/number/SayNumber.java

```diff
@@ -1,15 +1,17 @@
 package com.sayweb.number;
 
-import org.springframework.stereotype.Service;
+import java.util.HashMap;
+import java.util.Map;
 
-@Service
 public class SayNumber {
+  private static final long MIN_NUMBER = 0; // below range starting from 0
+  private static final long MAX_NUMBER = Long.MAX_VALUE; // upper range
```

Comment:
Make these two (min & max) public and reference them from AppController. 

There should be a single place only where such values are kept. Otherwise, we have to change values in future, we have to make changes in several places. Of course, it is not a problem in a small projects like this, but really risky in projects with millions of code with decades of legacy code. 

As a general rule, avoid duplicated code.